stl/inc/yvals_core.h

```diff
@@ -35,6 +35,7 @@
 // P0646R1 list/forward_list remove()/remove_if()/unique() Return size_type
 // P0653R2 to_address()
 // P0655R1 visit<R>()
+// P0738R2 istream_iterator Cleanup

```

Comment:
Hiss @StephanTLavavej for ruining my incredible proposal title! (No change requested.)

Reply:
Not everyone has the heart of a poet